README.md

```diff
@@ -52,6 +55,30 @@ amazon-managed-grafana-migrator -v
 amazon-managed-grafana-migrator discover --region eu-west-1
 ```
 
+### Migrating to Amazon Managed Grafana v10
+
+v9 and v10 introduced Grafana Service Accounts which will be required by the
+migrator, especially for v10. Note that Service Accounts are billed as active
+users
+
+1. Creating a Service Account
+
+```console
```

Comment:
Maybe you want to have an 'export' before this and grab the account id with jq. Theb just have this output be a bash environment variable. Then later on when the variable is used you can just say $SRC_SERVER_ACCOUNT_ID . That way people do not have to remember/scroll up and can just copy paste.